dal/RestDB.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import
+from tinydb import TinyDB, Query
+from dal.Tiny import Tiny
```

Comment:
@mansikulkarni96 Needs better name. Tiny is ambiguos. 
Let's make it baseDb since its an Abstract class

Reply:
renamed now thanks